cmd/kube-proxy/app/server.go

```diff
@@ -104,11 +112,15 @@ type Options struct {
 	master string
 	// healthzPort is the port to be used by the healthz server.
 	healthzPort int32
+
+	scheme *runtime.Scheme
```

Comment:
It's the only state we have, right? (next to globals) It looks strange in my eyes to have that in options. But I can live with it I guess.